### PR TITLE
[improve][doc] add managedLedgerDataReadPriority configuration description

### PR DESCRIPTION
**Pull request by ethqunzhong**
### Motivation
doc for https://github.com/apache/pulsar/pull/17803

### Documentation
Check the box below or label this PR directly.
Need to update docs? 
- [ ] `doc-required` 
(Your PR needs to update docs and you will update later)
- [ ] `doc-not-needed` 
(Please explain why)
- [x] `doc` 
(Your PR contains doc changes)
- [ ] `doc-complete`
(Docs have been already added)

### Matching PR in forked repository
PR in forked repository:
https://github.com/ethqunzhong/pulsar/pull/4

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@ethqunzhong from now on, the configuration docs are available at [Pulsar Reference](http://pulsar.apache.org/reference/#/) and generated automatically from code.

image

For how to update configuration docs, see https://pulsar.apache.org/contribute/documentation/contribution#update-pulsar-configuration-docs

**Comment by Anonymitaet** [comment]
cc @DaveDuggins @momo-jun

**Comment by Anonymitaet** [comment]
@hangc0276 could you please review this PR from a technical perspective? Thank you!

**Comment by ethqunzhong** [comment]
> @ethqunzhong from now on, the configuration docs are available at [Pulsar Reference](http://pulsar.apache.org/reference/#/) and generated automatically from code.
> 
> image
> 
> For how to update configuration docs, see https://pulsar.apache.org/contribute/documentation/contribution#update-pulsar-configuration-docs

thx for your advice. This is an efficient way to update pulsar configuration.

File: site2/website/versioned_docs/version-2.10.x/reference-configuration.md [hunk 1]
```diff
@@ -342,6 +342,7 @@ brokerServiceCompactionThresholdInBytes|If the estimated backlog size is greater
 |managedLedgerUnackedRangesOpenCacheSetEnabled|  Use Open Range-Set to cache unacknowledged messages |true|
 |managedLedgerOffloadDeletionLagMs|Delay between a ledger being successfully offloaded to long term storage and the ledger being deleted from bookkeeper | 14400000|
 |managedLedgerOffloadAutoTriggerSizeThresholdBytes|The number of bytes before triggering automatic offload to long term storage |-1 (disabled)|
+|managedLedgerDataReadPriority|Read priority when ledgers exists in both bookkeeper and the second layer storage, option values (tiered-storage-first, bookkeeper-first) |tiered-storage-first|
```
Comment (momo-jun) [inline site2/website/versioned_docs/version-2.10.x/reference-configuration.md:345]:
```suggestion
|managedLedgerDataReadPriority|Read priority when ledgers exist in both BookKeeper and the second layer storage. Option values are `tiered-storage-first` and `bookkeeper-first`. |tiered-storage-first|
```

Comment (ethqunzhong) [reply site2/website/versioned_docs/version-2.10.x/reference-configuration.md]: thx your suggestion.

**Review by momo-jun — APPROVED** [review]
LGTM.

**Comment by tisonkun** [comment]
Closing..

The source of docs are moved to http://github.com/apache/pulsar-site. Please open a PR there if you want to proceed this patch. And feel free to ping me for reviewing.

**Pull request closed without merging**